storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -676,13 +683,6 @@ func (self *ReconcileVolume) finalizeVolume(
 	saltenv string,
 ) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Volume.Name", volume.Name)
-
-	// Pending volume: can do nothing but wait for stabilization.
```

Comment:
Does this *need* to be removed, or would it make sense to keep this around for safety sake?

Reply:
We could keep it, it wouldn't be dangerous as it would be dead code.
But it would be redundant with the check done at the upper level, no?